api/query/cache/poll/poll.go

```diff
@@ -86,7 +86,7 @@ func KeepRunsUpdated(store shared.Datastore, logger shared.Logger, interval time
 }
 
 func wait(start time.Time, total time.Duration) {
-	t := total - time.Now().Sub(start)
+	t := total - time.Since(start)
```

Comment:
This is safe. As mentioned in the docs, this is equivalent. Which checker flagged this?